employee.html

```diff
@@ -0,0 +1,215 @@
+                    margin: 20px 0;
+                    margin-bottom: 0px;
+                }
+
+                /*----- Individual Tab Spacing -----*/
+                ul.tab-links {
+                  -webkit-padding-start: 4px;
+                }
+
+                div.basic-info {
+                  margin-bottom: 5px;
+                }
+
+                li.tab1 {
+                  margin-right: 5px;
+                }
+
+                li.tab2 {
+                  margin-right: 5px;
+                }
+
+                /*----- Tabs -----*/
+                .tabs {
+                    width:100%;
+                    display:inline-block;
+                    margin-top: 0px;
+                }
+
+                  /*----- Tab Links -----*/
+                  /* Clearfix */
+                  .tab-links:after {
+                      display:block;
+                      clear:both;
+                      content:'';
+                  }
+
+                  .tab-links li {
+                      float: left;
+                      list-style:none;
+                  }
+
+                  .tab-links a {
+                        text-decoration:  none;
+                        padding:9px 15px;
+                        display:block;
+                        border-radius:3px 3px 3px 3px;
+                        padding-left: 25px;
+                        padding-right: 25px;
+                        background:#1C1C1C;
+                        font-family: "lft-etica";
+                        font-weight: 500;
+                        font-size: 14px;
+                        color: white;
+                        transition:all linear 0.15s;
+                  }
+
+                  .tab-links a:hover {
+                      background:#8f362e;
+                      text-decoration:none;
+                  }
+
+                  li.active a, li.active a:hover {
+                      background:#fff;
+                      color:#4c4c4c;
+                  }
+                  .tab {
+                      display:none;
+                  }
+
+                  .tab.active {
+                      display:block;
+                  }
+
+                </style>
+
+                <div class="rating">
+                    <div class="stars">
+                    {% for star in stars %}
+                        <i id="star-{{forloop.counter}}" data-rating="{{forloop.counter}}" class="star fa
+                        {% if star = 'full' %}
+                            fa-star full
+                        {% elif star = 'half' %}
+                            fa-star-half-o half
+                        {% else %}
+                            fa-star-o empty
+                        {% endif %}"></i>
+                    {% endfor %}
+                    </div>
+                    <div class="message">
+                    {% if employee.num_ratings < 5 %}
+                        {{ ratings_needed }} more votes needed
```

Comment:
If this were phrased as
```html
{% if ratings_needed > 0 %}
    {{ ratings_needed }} more votes needed
```
then you could remove the magic constant `5`, which would further separate the frontend from the backend and allow that number to be changed in one place in the future, as opposed to two.